docs/SDK/Explanation/migration.md

```diff
@@ -5,218 +5,211 @@ sidebar_position: 2
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Migrating to V2
+# Migrating From V2 To V3
 
-`lit-js-sdk` is now considered deprecated and will only receive security updates. It is strongly recommended to migrate to the new **Lit JS SDK V2** for continued support and new features.
+The **Lit JS SDK V3** replaces the existing access control condition based encryption and JWT signing processes with new cryptographic primitives to offer a more secure and seamless user experience.
 
-The **Lit JS SDK V2** has been entirely revamped from the earlier `lit-js-sdk` and completely rewritten in TypeScript. This latest V2 SDK is much more modular and easier to use, and has a noticeably smaller bundle size.
+Refer to the [changelog](https://github.com/LIT-Protocol/js-sdk/blob/master/CHANGELOG.md) for more details.
 
-## Installing the V2 SDK
+## Per-Package Changes
 
-:::note
-You should use **at least Node v16.16.0** because of the need for the **webcrypto** library.
-:::
-
-Get started with **Lit JS SDK V2** by installing the package best suited for your environment.
-
-Browser & Node environments:
-
-```bash
-npm install @lit-protocol/lit-node-client
-```
-
-Node environment:
+### `@lit-protocol/bls-sdk`
 
-```bash
-npm install @lit-protocol/lit-node-client-nodejs
-```
-
-If you already have `lit-js-sdk` in your app, all you need to do is remove the old package and then update any import statement to reference the new SDK like so:
-
-```bash
-import * as LitJsSdk from '@lit-protocol/lit-node-client';
-```
+:::note
 
-If you are using TypeScript, be sure to install the `@lit-protocol/types` package. Check out the list of available packages [here]([https://github.com/LIT-Protocol/js-sdk/tree/master#packages](https://github.com/LIT-Protocol/js-sdk/tree/master#packages)).
+You should only use this package if you know what you are doing. Otherwise, please use `@lit-protocol/encryption` as a higher-level helper library.
 
-## Notable Changes
-
-**Lit JS SDK V2** includes some breaking changes from the latest version of the old `lit-js-sdk`.
+:::
 
-### Updating Imports
+#### Encryption
 
-Some methods have been moved to separate packages and must be accessed from those packages. Below are lists of methods underneath their respective packages.
+Previously in V2, you would have to:
 
- `@lit-protocol/crypto` 
+- Generate a symmetric key
+- Encrypt private data using this symmetric key
+- Encrypt this symmetric key using the BLS network key
+- Save this encrypted symmetric key with some access control conditions in the BLS network
 
-- decryptWithSymmetricKey
-- encryptWithSymmetricKey
-- importSymmetricKey
-- generateSymmetricKey
+Now in V3, you can perform client-side encryption using the `encrypt` method:
```

Comment:
"perform client-side encryption" is a bit vague compared to the list of steps mentioned above. "now in v3, all you would have to do is..."